### DISCUSSION TITLE
Havuz sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#291] [Zeka Sorusu]
Bir havuzu doldurmak üzere A, B ve C muslukları bulunmaktadır. (Evet evet havuz problemi bu)
Üç musluk birden açılırsa A nın tek başına dolduracağından 2 saat daha az, B nin tek başına dolduracağı zamandan 5 saat daha az, C nin tek başına dolduracağı sürenin ise yarısı kadar sürede havuz dolmaktadır.
A ve B musluklarının ikisi birden açılırsa havuz ne kadar sürede dolar?

**Comment by slos8888** [comment]
üç musluk birlikte açıldığında havuzun doldurulma süresi "2/c musluğunun saatteki doldurma kapasitesi" saat olacak.

**Reply by sonsuzus** [reply]
Yani? İşlemler, açıklamalar, sonuç? (saat cinsinden)
